02_bubble_sort/anishaNair/main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func bubble(n []int) []int {
+	for i := 0; i < len(n); i++ {
```

Comment:
Don't modify input slice, create a copy first.
Write a test ensuring that the input slice is not modified.